### ISSUE TITLE
Remember Color Scheme expanded/collapsed state

### ISSUE THREAD

**Issue by DAKarp** [SoftFever/OrcaSlicer#7813]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Color Scheme panel appears after every slice, covering up the preview each time

### Which printers will be beneficial to this feature?

All

### Describe the solution you'd like

If the color scheme panel was collapsed last time, remember this and show it collapsed next time too

**Comment by ilko-k** [comment]
Yes, I was about to post the same thing. I second this, it's very annoying to have that scheme displayed at each damn slice every single time a setting is changed.
